config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root 'application#temporary'
+  get 'upload/index'
+  get 'visualization/index'
+  get 'visualization/show'
+  get 'visualization/new'
+  get 'visualization/create'
```

Comment:
The `create` action does not need to be exposed as a route I don't think. The `new` action should be exposed instead but I'm not sure. I think the `create` action is what makes the http POST request when we create a new visualization.

Reply:
I thought we could just include the line `resources: visualization` in the place of lines 3-6 here because it automatically makes those routes/actions available when you do that? Not sure though

Reply:
I think we might be able to, but we don't have the visualization model / resource set up yet, so we just wanted to use this so we could make all the pages link to each other. It might work without having the model created, but we were afraid Rails might not like that so we didn't try it.

Reply:
One other thing - if we get rid of the create route, then we can't test it for it in the controller testing. Is that okay ? We can still call the action; the route is really only needed to link a controller action to a view, right ?